### DISCUSSION TITLE
How to disable by default / at launch?

### DISCUSSION THREAD

**Discussion by vietstone-ng** [ianyh/Amethyst#1590] [Q&A] [answered]
Currently, it's enabled at launch by default at launch time.
Is there a setting that I can turn it off? I mean I want it to launch as disabled by default. Then I go to enable it manually later.
![Screenshot 2024-01-11 at 18 25 07](https://github.com/ianyh/Amethyst/assets/744353/74be71b0-77f3-4160-abe5-a8257d8ce5ca)

**Comment by ianyh** [comment, marked as answer]
There is no way to do it currently, but the feature request is tracked in #828

**Comment by lemonbottle** [comment]
Just curious. Is this a huge issue? Would it not suffice to just disable Start at login, and press CMD+SPACE then start typing "Amethyst" to quickly launch when you need it?
